slices/dotnet-sdk-aot-9.0.yaml

```
@@ -0,0 +1,13 @@
+package: dotnet-sdk-aot-9.0
+
+essential:
+  - dotnet-sdk-aot-9.0_copyright
+
+slices:
+  libs:
+    contents:
```

```suggestion
    essential:
      - dotnet-sdk-9.0_core
      - libbrotli1_libs
      - libc6_libs
      - libgcc-s1_libs
      - libssl3t64_libs
      - libstdc++6_libs
      - zlib1g_libs
    contents:
```

Or if these aren't always needed for the slice, a description of why and when to add them would be useful.

---

Fixed with 862342c700f81427b847473d6b940b5549b219dd